### DISCUSSION TITLE
Exception when connecting to rp.

### DISCUSSION THREAD

**Discussion by jackschrott** [linien-org/linien#286] [Q&A] [answered]
Hello,

I had the standalone binary (linien-windows-0.5.3.exe) running well out of the downloads folder of my Windows 10 desktop. I was able to connect to my rp fine. Now after having unplugged my rp and also moved the executable out of Downloads (I've tried moving it both to Desktop and a folder I have at  C:\Users\me\Software) I am no longer able to connect to the rp. When I try to connect I get the following "Exception occurred when connecting to device". I have read issue #283 and when I ssh into my rp and do screen -ls, I see that the linien server is not running. 
```
root@rp-xxxxxx:~# screen -ls
No Sockets found in /var/run/screen/S-root.
```
Any advice would be greatly appreciated

**Comment by jackschrott** [comment, marked as answer]
I forgot that I had started running openssh ssh server and openssh authentication agent. Stopping both of these by going to Services let me reconnect

**Reply by bleykauf** [reply]
Thanks for sharing your solution, open an issue to mention possible issues with this in the documentation #296 .
